### DISCUSSION TITLE
Restriccion de tiempo

### DISCUSSION THREAD

**Discussion by ncruzm** [IIC2133-PUC/2024-2#159] [Tarea 2] [answered]
Hola! Con la restricción de tiempo de 5 segundos se refiere a el tiempo para el usuario o para el sistema, porque cuando imprimo el tiempo de ejecución con el comando time me dice:

./secretfind tests/hard-2/input.txt output.txt  4,71s user 0,30s system 90% cpu 5,523 total en ese caso que se considera?

**Comment by tomastrivino** [comment, marked as answer]
el total

**Reply by catalinacifuentes** [reply]
![Captura de Pantalla 2024-10-28 a la(s) 09 32 33](https://github.com/user-attachments/assets/a033713e-7073-4290-8748-234c88d252ef)
En este caso entonces sería user + sys el tiempo que se considera, no el que dice real?

**Reply by tomastrivino** [reply]
El real! Sorry!
